pkg/ensurance/runtime/runtime.go

```diff
@@ -10,42 +10,32 @@ import (
 	gprcconnection "github.com/gocrane/crane/pkg/ensurance/grpc"
 )
 
-// runtimeEndpointIsSet is true when RuntimeEndpoint is configured
 // runtimeEndpoint is CRI server runtime endpoint
-func getRuntimeClientConnection(runtimeEndpoint string, runtimeEndpointIsSet bool) (*grpc.ClientConn, error) {
-
-	if runtimeEndpointIsSet && runtimeEndpoint == "" {
-		return nil, fmt.Errorf("runtime-endpoint is not set")
-	}
-
-	if !runtimeEndpointIsSet {
-		klog.V(2).Infof("Runtime connect using default endpoints: %v. You can set the endpoint instead.", defaultRuntimeEndpoints)
-		return gprcconnection.InitGrpcConnection(defaultRuntimeEndpoints)
+func getRuntimeClientConnection(runtimeEndpoint string) (*grpc.ClientConn, error) {
+	var runtimeEndpoints []string
+	if runtimeEndpoint != "" {
+		runtimeEndpoints = append(runtimeEndpoints, runtimeEndpoint)
 	}
-
-	return gprcconnection.InitGrpcConnection([]string{runtimeEndpoint})
+	runtimeEndpoints = append(runtimeEndpoints, defaultRuntimeEndpoints...)
```

Comment:
Add the k3s runtime endpoint to defaultRuntimeEndpoints.
https://github.com/gocrane/crane/blob/76d56448d3fd23614469d722f467881a5cfa68bc/pkg/ensurance/runtime/runtime_unix.go#L6